libcxx/test/std/ranges/range.adaptors/range.zip/iterator/increment.pass.cpp

```diff
@@ -59,7 +59,7 @@ constexpr bool test() {
 
   {
     //  bidi
-    int buffer[2] = {1, 2};
+    int buffer[3] = {1, 2, 3};
```

Comment:
Can you explain the need for these changes?

Reply:
It was doing an out-of-bounds read, which was not previously diagnosed

Reply:
Well, the test is called `increment.pass.cpp`, and used to be valid, so I assume it should remain valid!

Reply:
I guess the out of bounds read was in line 79? I think we should rather fix getting the address. Right now we'd not iterate through the whole range properly otherwise.

Reply:
Ah, shit. You can't actually get the iterator. I think this is fine as-is.